### ISSUE TITLE
cli: include sysctl and netstat in `debug.zip`

### ISSUE THREAD

**Issue by erikgrinaker** [cockroachdb/cockroach#102125]
It would be useful to include in `debug.zip` the output of `sysctl -a` and `netstat -s` on every node, to inspect e.g. kernel TCP settings and metrics when debugging network issues.

**Comment by tbg** [comment]
or better, add some of this to metrics so we have history and can work with the data better. Lots of numbers are in procfs

```
ubuntu@grinaker-231-0001:~$ awk '$1 ~ "Tcp:" { print $13 }' /proc/net/snmp
RetransSegs
906466
```

**Comment by tbg** [comment]
Wouldn't be surprised if `gosigar` and these other kinds of libraries already picked most of these up

**Comment by erikgrinaker** [comment]
Yes, even better, but there's a ton of OS metrics and I don't know if we want them clogging up the time series database. Maybe we can pick out a few particularly important ones.

**Comment by tbg** [comment]
Right, didn't mean to scrape everything under the sun, just that we mostly just have to find a library that has what we want and hook up the metrics we need. Not much reinventing the wheel should be needed here.

**Comment by irfansharif** [comment]
Some internal discussion [here](https://cockroachlabs.slack.com/archives/C01C1HGC46Q/p1686935437076659?thread_ts=1686932805.130289&cid=C01C1HGC46Q). I'm going to re-title this issue to include more networking metrics+diagnostics info.
